python_utils.py

```diff
@@ -464,3 +464,18 @@ def reraise_exception():
     # the stacktrace. See https://stackoverflow.com/a/18188660/3688189.
     exec_info = sys.exc_info()
     six.reraise(exec_info[0], exec_info[1], tb=exec_info[2])
+
+
+def reload_module(mod):
```

Comment:
No, don't do this. Let me try to take a look at where you use it to see if we can get around it.

Reply:
Obsolete.